frontend.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": [
+      "github>empathyco/.github",
+      ":semanticCommitTypeAll(chore)",
+      "schedule:earlyMondays"
+    ],
+    "rangeStrategy": "bump",
+    "ignoreDeps": ["npm", "node", "@types/node"]
```

Comment:
```suggestion
    "ignoreDeps": ["npm", "node"]
```
major `@types/node` should match `node` version but there are minor and fixes.
I think that renovate does not suggest v22 of types if we are using node 20